profiles/common/modules/features/newsletters/newsletters.install.inc

```diff
@@ -0,0 +1,375 @@
+<?php
+
+/**
+ * @file
+ * Code for the newsletters module.
+ */
+
+/**
+ * Soft configure simplenews content type.
+ */
+function _newsletters_install_soft_config_simplenews() {
+  // Use soft config for newsletters comment form location.
+  multisite_config_service('comment')->setThreadingCommentForContentType('simplenews', 1);
+
+  $t = get_t();
+  // Create new fields.
+  // Data taken from newsletters.features.field_base.inc.
+  $fields = array(
+    'field_related_contents'  => array(
+      'active' => 1,
+      'cardinality' => -1,
+      'deleted' => 0,
+      'entity_types' => array(),
+      'field_name' => 'field_related_contents',
+      'foreign keys' => array(
+        'node' => array(
+          'columns' => array(
+            'target_id' => 'nid',
+          ),
+          'table' => 'node',
+        ),
+      ),
+      'indexes' => array(
+        'target_id' => array(
+          0 => 'target_id',
+        ),
+      ),
+      'locked' => 1,
```

Comment:
@aritomelo the procedure requires all fields are locked. See the checklist of an old feature here https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/ReadSpeaker+checklist 
This is because, after we deploy a major version, devops perform a feature revert all (except on cce_basic_config)

Reply:
@Fefaine Thx for your feedback.